mm2src/coins_activation/src/prelude.rs

```diff
@@ -21,6 +23,12 @@ impl TxHistory for UtxoActivationParams {
     fn tx_history(&self) -> bool { self.tx_history }
 }
 
+#[cfg(feature = "enable-sia")]
+impl TxHistory for SiaCoinActivationParams {
+    fn tx_history(&self) -> bool { self.tx_history }
+}
+
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Good catch, thank you. 

This was previously in dev branch and recently removed. It's included here because of a dev->sia-dev rebase. Taking a look now on how to remedy it and taking the time to ensure no other similar issues were introduced.